indico/web/forms/fields/principals.py

```diff
@@ -76,7 +81,8 @@ def __init__(self, *args, **kwargs):
     def _convert_principal(self, principal):
         return principal_from_fossil(principal, allow_pending=self.allow_external,
                                      allow_emails=self.allow_emails, allow_networks=self.allow_networks,
-                                     existing_data=self.object_data, event=self._event)
+                                     existing_data=self.object_data, event=self._event,
+                                     allow_category_roles=self.allow_category_roles)
```

Comment:
this change can go away as well (it breaks the field in its current state)

Reply:
oh I missed it